plugins/rebalance/api/req_resp.go

```diff
@@ -0,0 +1,77 @@
+package rebalance
+
+import (
+	"github.com/pborman/uuid"
+)
+
+// Status represents Rebalance Status
+type Status uint64
+
+const (
+	// NotStarted should be set only for a node in which rebalance process is not started
+	NotStarted Status = iota
+	// Started should be set only for a node that has been just started rebalance process
+	Started
+	// Stopped should be set only for a node that has been just stopped rebalance process
+	Stopped
+	// Complete should be set only for a node that the rebalance process is completed
+	Complete
+	// Failed should be set only for a node that are failed to run rebalance process
+	Failed
+)
+
+// Command represents Rebalance Commands
+type Command uint64
+
+const (
+	// CmdNone should be set only when given cmd is none
+	CmdNone Command = iota
+	// CmdStart should be set only when given cmd is rebalance start
+	CmdStart
+	// CmdStop should be set only when given cmd is rebalance stop
+	CmdStop
+	// CmdStatus should be set only when given cmd is rebalance status
+	CmdStatus
+	// CmdFixlayoutStart should be set only when given cmd is rebalance fix-layout start
+	CmdFixlayoutStart
+	// CmdStartForce should be set only when given cmd is rebalance start force
+	CmdStartForce
+)
+
+// NodeInfo represents Node information needed to store
+type RebalNodeStatus struct {
+	NodeID            uuid.UUID
+	Status            string  // This is per Node Rebalance daemon Status
+	RebalancedFiles   string
```

Comment:
Just wondering why all members are strings? Wouldn't it be better for these to use relevant types?

Reply:
NVM. I got it. Because this is how the xdr dict contains them.
But it would still be preferrable to convert to proper types when sending them to the ReST clients. JSON handles types much better than our dicts.

Reply:
I would prefer proper types as well but could not get them to work with the xdr. If you can point me to how to do this, I can make the changes.